src/distro/distro.py

```diff
@@ -1301,10 +1349,12 @@ def _parse_distro_release_file(self, filepath: str) -> Dict[str, str]:
             A dictionary containing all information items.
         """
         try:
-            with open(filepath, encoding="utf-8") as fp:
+            with open(
+                self.__resolve_chroot_symlink(filepath), encoding="utf-8"
+            ) as release_file:
```

Comment:
(I would personally revert `release_file` back to `fp` to reduce "noise" in the diff, but just my 2 cents, please feel free to ignore.)